korgan/client_document_feedback_hotfix.py

```
@@ -0,0 +1,334 @@
+    )
+    return text + tail
+
+
+def progress_text(kind: str, language: str = "ru") -> str:
+    """Return a visible status message for an actually authorized generation."""
+    if language == "kk":
+        return "⏳ Құжат өңделуде. KORGAN қолданылатын құқықты, талаптарды және дәлелдерді тексеріп, Word-файлды қалыптастырып жатыр. Күрделі іс бірнеше минут алуы мүмкін."
+    return "⏳ Документ в работе. KORGAN проверяет применимое право, требования и доказательства и формирует Word-файл. По сложному делу это может занять несколько минут."
+
+
+async def send_checklist_once(message: Any, state: Any, kind: str) -> bool:
```

Ruff (0.16.2)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `message`

(ANN401)